package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@folio/stripes-webpack",
-  "version": "3.0.0",
+  "version": "3.0.2",
```

Comment:
Why was the version bumped here? Since we haven't even formally released `3.0.0`, this should not have changed.